src/wasm-linker.h

```diff
@@ -241,7 +244,7 @@ class Linker {
 
   // Support for emscripten integration: generates dyncall thunks, emits
   // metadata for asmConsts, staticBump and initializer functions.
-  void emscriptenGlue(std::ostream& o);
+  void emscriptenGlue(std::ostream& o, bool allowMemoryGrowth);
```

Comment:
I'm generally opposed to boolean arguments, but was thinking that in this case it had a 1:1 mapping with user input, i.e. the "--allow-memory-growth" flag. However that's more of a consequence of how it *happens* to work, today, so it makes sense to change it.

Going to go with calling it directly from s2wasm.cpp, that seems like the right way to slice it.
Thanks!